x-pack/plugins/apm/server/tutorial/index.ts

```diff
@@ -14,6 +14,9 @@ import {
   TutorialsCategory
 } from '../../../../../src/plugins/home/server';
 
+// duplicated in x-pack/plugins/apm/common/index_pattern_constants.ts
+const APM_STATIC_INDEX_PATTERN_ID = 'apm_static_index_pattern_id';
```

Comment:
why haven't you imported the one in `x-pack/plugins/apm/common/index_pattern_constants.ts`?

Reply:
Good point 👍 
(I was still in the old mindset that this code lived outside the APM app)